ecs-cli/integ/e2e/local_test.go

```diff
@@ -85,6 +86,82 @@ func TestECSLocal(t *testing.T) {
 						})
 					},
 				},
+				{
+					args: []string{"local", "down", "--all"},
+					execute: func(t *testing.T, args []string) {
+
+						stdout, err := integ.RunCmd(t, args)
+						require.NoError(t, err)
+						stdout.TestHasAllSubstrings(t, []string{
+							"No running ECS local tasks found",
+						})
+					},
+				},
+				{
+					args: []string{"local", "down", "-f", "task-definition.json"},
+					execute: func(t *testing.T, args []string) {
+						stdout, err := integ.RunCmd(t, args)
+						require.NoError(t, err)
+						stdout.TestHasAllSubstrings(t, []string{
+							"Searching for containers from local file task-definition.json",
+							"No running ECS local tasks found",
+						})
+					},
+				},
+			},
+		},
+		"from task def": {
+			sequence: []commandTest{
+				{
+					args: []string{"local", "create", "-f", ""},
+					execute: func(t *testing.T, args []string) {
+						tempFileName := createLocalTaskDefFile(t)
+						args[3] = tempFileName
+						stdout, err := integ.RunCmd(t, args)
+						require.NoError(t, err)
+						stdout.TestHasAllSubstrings(t, []string{
+							"Successfully wrote docker-compose",
+						})
+						os.Remove(tempFileName)
+						os.Remove("docker-compose.ecs-local.yml")
```

Comment:
oh, really good idea. i'll think about how to do that in the next PR. I have some more integ tests to write so that's a good addition.

Reply:
Really good job! Just a random thought, for `from task def` part, can we also test up/ps/down without `-f` like using the default `task-definition.json`?

Reply:
I think that's a really good idea, and it's on my list of other integ tests to write. For these, I just wanted to use the temp file function in go so that it was clear that they were auto-generated by tests. At least one test in the next batch will use the default option, though.

Reply:
re: @iamhopaul123 's suggestion - I like that we're using temp files for these and would like to continue doing that, however I see the benefit of testing the default behavior. I'd like to suggest that if we do add such a test we just check in a canonical `task-definition.json` we know the contents of so we don't have to worry about creating/destroying a "real" file during the course of the tests. Thoughts?

Reply:
however... do we know for sure from which directory the test will run + look for the file? it is it `/e2e`?

Reply:
Running make integ-test locally, all temp files are created in `e2e` but I'm not sure how to ensure that. Is there an environment variable anywhere (Makefile, perhaps?) that specifies the /amazon-ecs-cli directory? We could specify longer paths for the tempfiles to ensure that they're always created in a deterministic place. But that may not be necessary if the working directory for `make integ-test` is always deterministic.

Reply:
what I mean is if we want to test the default behavior (i.e., not using `-f`) the `task-definition.json` file needs to be in the current working directory of the cmd being executed, and I'm not 100% sure this is `/e2e`. If it turns out we have to add a `task-definition.json` file to, say, `/ecs-cli` (where `main.go` is) that would be really weird and i would vote against it. (but in `/e2e` is more sane)